api/cpp/esp-idf/slint/include/slint-esp.h

```diff
@@ -44,18 +45,24 @@ struct SlintPlatformConfiguration
     esp_lcd_touch_handle_t touch_handle = nullptr;
     /// The buffer Slint will render into. It must have have the size of at least one frame. Slint
     /// calls esp_lcd_panel_draw_bitmap to flush the buffer to the screen.
-    std::optional<std::span<slint::platform::Rgb565Pixel>> buffer1 = {};
+    std::optional<std::span<PixelType>> buffer1 = {};
     /// If specified, this is a second buffer that will be used for double-buffering. Use this if
     /// your LCD panel supports double buffering: Call `esp_lcd_rgb_panel_get_frame_buffer` to
     /// obtain two buffers and set `buffer` and `buffer2` in this data structure.
-    std::optional<std::span<slint::platform::Rgb565Pixel>> buffer2 = {};
+    std::optional<std::span<PixelType>> buffer2 = {};
     slint::platform::SoftwareRenderer::RenderingRotation rotation =
             slint::platform::SoftwareRenderer::RenderingRotation::NoRotation;
-    /// Swap the 2 bytes of RGB 565 pixels before sending to the display. Use this
-    /// if your CPU is little endian but the display expects big-endian.
-    bool color_swap_16 = false;
+    /// Swap the 2 bytes of RGB 565 pixels before sending to the display, or turn RGB into BGR. Use
```

Comment:
```suggestion
    /// Swap the 2 bytes of RGB 565 pixels before sending to the display, or turn 24bit RGB into BGR. Use
```

We don't generate documentation for this, right?
If we do, I wonder if this documentation shouldn't need to go to the new name.
But in the header it looks better like this, that's true.

Reply:
Right, we don't generate documentation for this at all yet. It's not integrated into any of your docs :(